docs/examples/101/1vm-2nics-2subnets-1vnet/main.json

```diff
@@ -231,5 +231,12 @@
       "type": "string",
       "value": "[reference(resourceId('Microsoft.Network/publicIPAddresses', variables('publicIPAddressName'))).ipAddress]"
     }
+  },
+  "metadata": {
+    "_generator": {
+      "name": "Bicep.Core",
+      "version": "0.2.418.59698",
```

Comment:
>version [](start = 7, length = 7)

@anthony-c-martin called this out on teams. This number will change with every build and every PR. I think we need to omit it or set it to a fake value in this repo. When samples move to quickstart, then we would switch to a real version.